test/test_terraform.py

```diff
@@ -30,12 +31,14 @@
  ]
 
 CMD_CASES = [
-    ['method', 'expected_output', 'expected_ret_code', 'expected_logs', 'folder'],
+    ['method', 'expected_output', 'expected_ret_code', 'expected_exception', 'expected_logs', 'folder'],
     [
         [
             lambda x: x.cmd('plan', 'var_to_output', no_color=IsFlagged, var={'test_var': 'test'}) ,
-            "doesn't need to do anything",
+            ["doesn't need to do anything",
+             "no\nactions need to be performed"],
             0,
```

Comment:
please help add one comment here to tell which string belong to which version.
We can try to figure out version compatibility on test script with other ticket.

Reply:
Will do, thanks